rotkehlchen/tests/conftest.py

```diff
@@ -218,6 +218,20 @@ def beaconchain_matcher(r1, r2):
 
         return r1.uri == r2.uri and r1.method == r2.method  # normal check
 
+    def alchemy_api_matcher(r1, r2):
+        """Match Alchemy price API paths, ignoring API key."""
+        if (
+            (base_url := 'https://api.g.alchemy.com/prices/v1/') and
+            r1.uri.startswith(base_url) and
+            r2.uri.startswith(base_url)
+        ):
+            # Extract everything after API key by finding second '/' after base_url
+            path1 = r1.uri[r1.uri.find('/', len(base_url)) + 1:]
+            path2 = r2.uri[r2.uri.find('/', len(base_url)) + 1:]
+            return path1 == path2 and r1.method == r2.method
+        return r1.uri == r2.uri and r1.method == r2.method
```

Comment:
```suggestion
    def alchemy_api_matcher(r1, r2):
        """Match Alchemy price API paths, ignoring API key."""

        if (
            r1.uri.startswith('https://api.g.alchemy.com/prices/v1/') and
            r2.uri.startswith('https://api.g.alchemy.com/prices/v1/')
        ):
            # Extract everything after API key by finding second '/' after base_url
            path1 = r1.uri[r1.uri.find('/', len(base_url)) + 1:]
            path2 = r2.uri[r2.uri.find('/', len(base_url)) + 1:]
            return path1 == path2 and r1.method == r2.method
        return r1.uri == r2.uri and r1.method == r2.method
```

hardcode it since it is faster than assigning a variable

Reply:
it's used in the lines below, to define `path1` and `path2`.